cmd/snap/cmd_info.go

```diff
@@ -163,27 +163,68 @@ func coalesce(snaps ...*client.Snap) *client.Snap {
 	return nil
 }
 
-// formatDescr formats a given string (typically a snap description)
+func rTrimRightSpace(text []rune) []rune {
+	j := len(text)
+	for j > 0 && unicode.IsSpace(text[j-1]) {
+		j--
+	}
+	return text[:j]
+}
+
+func rLastIndexSpace(a []rune) int {
```

Comment:
What is this function supposed to do? It returns the index of the first space character it can find, going right to left.

Reply:
If that's the case, name is inverted: lastSpaceIndex

Reply:
In both `strings` and `bytes` there's already a family of `LastIndexFoo` functions, of which this is a silly extension (except it's for runes).

That's also the reason I prefixed them with something, so the name would better match the `strings` and `bytes` families, of which again these are siblings.

My hope was that if you saw `runesLastIndexSpace` being called it would be immediately obvious what it did and it wouldn't need more explanation.

Reply:
@chipaca Agreed, consistency wins, and yes, the current function seems clean and fine, except I'm not sure we need it for different reasons per the actual review.